src/components/Dashboard/AccountBanner.js

```diff
@@ -85,8 +83,6 @@ function AccountBanner({ status, loading, minActiveBalance, activeBalance }) {
     theme
   )
 
-  if (loading) return <Loading height={48} />
```

Comment:
I think we should keep a loading ring here because otherwise it's going to show an incorrect state to the juror every time he/she loads the dashboard